internal/http/services/owncloud/ocs/response/response.go

```diff
@@ -155,7 +156,7 @@ func WriteOCSResponse(w http.ResponseWriter, r *http.Request, res Response, err
 	version := APIVersion(r.Context())
 	m := statusCodeMapper(version)
 	statusCode := m(res.OCS.Meta)
-	if version == "v2.php" && statusCode == http.StatusOK {
+	if version == "2" && statusCode == http.StatusOK {
```

Comment:
is this expected?

Reply:
Yeah, at least that is how the counterpart on ocis behave(s|d). From https://github.com/owncloud/ocis/blob/master/ocs/pkg/service/v0/response/response.go#L77 : 

```
    if version == ocsVersion2 && statusCode == http.StatusOK {
        rsp.OCS.Meta.StatusCode = statusCode
    }
    return nil
```
(where ocsVersion2 is defined as `"2"`)

Reply:
yep ocs v2 maps the application level status codes to transport level status codes: the ocs ok status will become a 200 Ok HTTP status.